### DISCUSSION TITLE
How to SWAP in Mojo

### DISCUSSION THREAD

**Discussion by Sai-Vignesh** [modular/modular#312] [Q&A] [answered]
Unlike Python, we can't swap the values of variables in a single statement.
like `a,b = b,a`

**Comment by lattner** [comment, marked as answer]
Thanks for the nudge, I implemented support for tuples on the left side.  They still require parens, but this allows `(a, b) = (b, a)` to work in the next update.  In the meantime you can use a temporary like:

```
tmp = a
a = b
b = tmp
```
